src/main/java/eu/europa/ted/eforms/sdk/domain/xml/CodeList.java

```diff
@@ -0,0 +1,118 @@
+package eu.europa.ted.eforms.sdk.domain.xml;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+
+/**
+ * <p>Java class for anonymous complex type.
```

Comment:
This comment is not really useful, I guess it was autogenerated.
Same remark for the other comments in this file for the getters and setters, and for the other added files in this folder.